Started by user [8mha:////4K7SonDTAuGL7kSjRxIwkgedPGlCQunhdvEn8vXX5ES2AAAAmR+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAy+EgZe/dLi1CL94qTUjNSiRAAwkGaUwgAAAA==[0mSwadesh Behera
Running as SYSTEM
[EnvInject] - Loading node environment variables.
Building in workspace /var/jenkins_home/workspace/01-fdc-standard-manifest-processing-job
The recommended git tool is: NONE
using credential vinay-github
 > git rev-parse --resolve-git-dir /var/jenkins_home/workspace/01-fdc-standard-manifest-processing-job/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/The-Frontdoor-Collective/fdc-etl.git # timeout=10
Fetching upstream changes from https://github.com/The-Frontdoor-Collective/fdc-etl.git
 > git --version # timeout=10
 > git --version # 'git version 2.30.2'
using GIT_ASKPASS to set credentials vinay-github
 > git fetch --tags --force --progress -- https://github.com/The-Frontdoor-Collective/fdc-etl.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse refs/remotes/origin/develop^{commit} # timeout=10
Checking out Revision 0336683137a7fc1120047307b30697ca144f6735 (refs/remotes/origin/develop)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 0336683137a7fc1120047307b30697ca144f6735 # timeout=10
Commit message: "Merge pull request #13 from The-Frontdoor-Collective/feature/ITMF-27"
 > git rev-list --no-walk 0336683137a7fc1120047307b30697ca144f6735 # timeout=10
[01-fdc-standard-manifest-processing-job] $ /bin/sh -xe /tmp/jenkins3472560697358137557.sh
+ cd python_scripts
+ [ dev = dev ]
+ python3 standard_client_manifest.py 121
Socket exception: Connection reset by peer (104)
Connection successfully established ... 
Found few files and started loading
f76565e9-7283-4ca6-8306-788eb0cd4bb8_20220825170322.csv Processing In-Progress
Manifest f76565e9-7283-4ca6-8306-788eb0cd4bb8_20220825170322.csv have all mandatory fields, will be loaded 
1 record inserted
Traceback (most recent call last):
  File "/var/jenkins_home/workspace/01-fdc-standard-manifest-processing-job/python_scripts/standard_client_manifest.py", line 174, in <module>
    sftp.remove(remoteFilePath)
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/pysftp/__init__.py", line 728, in remove
    self._sftp.remove(remotefile)
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/paramiko/sftp_client.py", line 398, in remove
    self._request(CMD_REMOVE, path)
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/paramiko/sftp_client.py", line 821, in _request
    num = self._async_request(type(None), t, *arg)
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/paramiko/sftp_client.py", line 846, in _async_request
    self._send_packet(t, msg)
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/paramiko/sftp.py", line 198, in _send_packet
    self._write_all(out)
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/paramiko/sftp.py", line 162, in _write_all
    n = self.sock.send(out)
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/paramiko/channel.py", line 801, in send
    return self._send(s, m)
  File "/var/jenkins_home/.local/lib/python3.9/site-packages/paramiko/channel.py", line 1198, in _send
    raise socket.error("Socket is closed")
OSError: Socket is closed
Build step 'Execute shell' marked build as failure
[Office365connector] Matched status 'FAILURE' for webhook with name 'null'.
Finished: FAILURE
